packages/components/src/ui/radix-dropdown/stories/index.tsx

```diff
@@ -0,0 +1,241 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS } from '../../../utils';
+import {
+	DropdownMenu,
+	DropdownMenuItem,
+	DropdownSubMenu,
+	DropdownMenuSeparator,
+	DropdownMenuCheckboxItem,
+	DropdownMenuGroup,
+	DropdownMenuLabel,
+	DropdownMenuRadioGroup,
+	DropdownMenuRadioItem,
+	DropdownSubMenuTrigger,
+} from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, createContext, useContext } from '@wordpress/element';
+import { menu, wordpress } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import Icon from '../../../icon';
+
+const DropdownMenuStoryContext = createContext< {
+	bookmarksChecked?: boolean;
+	setBookmarksChecked?: ( v: boolean ) => void;
+	urlsChecked?: boolean;
+	setUrlsChecked?: ( v: boolean ) => void;
+	person?: string;
+	setPerson?: ( v: string ) => void;
+} >( {} );
+
+const meta: ComponentMeta< typeof DropdownMenu > = {
+	title: 'Components/Radix DropdownMenu',
+	component: DropdownMenu,
+	subcomponents: {
+		DropdownMenuItem,
+		DropdownSubMenu,
+		DropdownSubMenuTrigger,
+		DropdownMenuSeparator,
+		DropdownMenuCheckboxItem,
+		DropdownMenuGroup,
+		DropdownMenuLabel,
+		DropdownMenuRadioGroup,
+		DropdownMenuRadioItem,
+	},
+	argTypes: {
+		children: { control: { type: null } },
+		trigger: { control: { type: null } },
```

Comment:
There is currently a console warning generated by Storybook, which seems caused by the React components passed in the `children` and `trigger` props being Emotion styled components:

```
We've detected a cycle in arg 'components-radix-dropdownmenu--default.trigger'. Args should be JSON-serializable.

Consider using the mapping feature or fully custom args:
- Mapping: https://storybook.js.org/docs/react/writing-stories/args#mapping-to-complex-arg-values
- Custom args: https://storybook.js.org/docs/react/essentials/controls#fully-custom-args
```